docs/hive-integration/crds/clusterImageSet.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: hive.openshift.io/v1
 kind: ClusterImageSet
 metadata:
-  name: openshift-v4.9.0
+  name: openshift-v4.15.0
```

Comment:
Why update this now? Everything kubeapi related should remain the same, right?

Reply:
OK I remember now, this is because of `agentBasedInstaller` test that uses it, which uses restAPI